src/spartan_tcp_handler.erl

```diff
@@ -39,7 +39,6 @@ callback_mode() ->
     state_functions.
 
 init([Ref, Socket, Transport, Opts]) ->
-    {ok, _} = timer:kill_after(?TIMEOUT * 5),
```

Comment:
This timeout was added because of https://jira.mesosphere.com/browse/DCOS-5858. This timeout predates the other timeouts. Maybe it is no more required but we should keep an eye.